draft-ietf-httpbis-header-structure.md

```diff
@@ -305,6 +305,10 @@ For example:
 Example-IntegerHeader: 42
 ~~~
 
+Parsers MUST accurately support integers in the range -9,007,199,254,740,991 to
+9,007,199,254,740,991 inclusive (i.e., -2^53+1 to 2^53-1) at a minimum. Parsers MUST fail parsing
+on values not within the range supported accurately.
```

Comment:
This reads a little awkward to me. Perhaps something like "Implementations must have a range for supported integers. At a minimum this range must be X to Y. Parsers must fail on parsing values outside the range for supported integers."